Sources/funnet/Core/NetworkResponder.swift

```diff
@@ -9,19 +9,23 @@ import Foundation
 
 public protocol NetworkResponderProtocol {
     var taskHandler: (URLSessionDataTask?) -> Void { get set }
+    var downloadTaskHandler: (URLSessionDownloadTask?) -> Void { get set }
     var responseHandler: (URLResponse?) -> Void { get set }
     var httpResponseHandler: (HTTPURLResponse) -> Void { get set }
     var dataHandler: (Data?) -> Void { get set }
+    var urlHandler: (URL?) -> Void { get set }
```

Comment:
Is this the URL to the file on the system? If so, I think we need a clearer name, I though this was the request URL for like 10m